src/store/authors/slice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { fetchAuthors, addAuthor } from './thunk';
+
+export const initialState = { authors: [] };
+
+const authorsSlice = createSlice({
+	name: 'authors',
+	initialState,
+	extraReducers: (builder) => {
+		builder
+			.addCase(fetchAuthors.fulfilled, (state, action) => {
+				state.authors = action.payload;
+			})
+			.addCase(addAuthor.fulfilled, (state, action) => {
```

Comment:
A common practice to use the same prefix name, which API is using. E.g.: `getSmth()', 'createSmth()' and so on.